proto/process/agent.proto

```diff
@@ -24,6 +24,7 @@ message ResCollector {
 
 message CollectorProc {
 	string  hostName = 2;
+	string networkId = 11;
```

Comment:
Any reason we can't have this in `SystemInfo`?

Reply:
Not in particular. I will say that we don't send systemInfo with CollectorConnections (just CollectorProc and CollectorContainer) so it would be a little bit inconsistent.

Reply:
Fair - a bit big of an object for just one field as well.